core/mono-samples/iOS/Program.csproj

```
@@ -0,0 +1,55 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <RuntimeIdentifier>ios-x64</RuntimeIdentifier>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Runtime.iOS.Sample.Mono" Version="6.0.0-dev" />
+  </ItemGroup>
+
+  <UsingTask TaskName="AppleAppBuilderTask"
+             AssemblyFile="$(NuGetPackageRoot)microsoft.net.runtime.ios.sample.mono/6.0.0-dev/tools/net6.0/AppleAppBuilder.dll" />
```

Instead of using `NuGetPackageRoot` we should set `GeneratePathProperty="true"` on the `<PackageReference>`, that will set a property called `$(PkgMicrosoft_NET_Runtime_iOS_Sample_Mono)` which points to the directory:

```suggestion
             AssemblyFile="$(PkgMicrosoft_NET_Runtime_iOS_Sample_Mono)\tools\net6.0\AppleAppBuilder.dll" />
```

---

Thanks!